content/en/news/releases/1.13.x/announcing-1.13.9/index.md

```diff
@@ -7,7 +7,13 @@ publishdate: 2022-10-11
 release: 1.13.9
 ---
 
-This release contains a patch for [CVE-2022-41715](https://github.com/golang/go/issues/55949) and bug fixes to improve robustness. This release note describes what is different between Istio 1.13.8 and Istio 1.13.9.
+This release contains a fix for [CVE-2022-39278](/news/security/istio-security-2022-007/#cve-2022-39278) and
+bug fixes to improve robustness. We recommend users install this release instead of Istio 1.13.8,
+which does not contain the above CVE fix.
+This release note describes what is different between Istio 1.13.8 and Istio 1.13.9.
+
+FYI, This release includes security fixes in Go 1.18.7 (released 2022-10-04) for the `archive/tar`, `net/http/httputil`, and `regexp` packages.
```

Comment:
I believe 1.13 still uses Go 1.17, but regexp was imported from an external library that addressed the CVE https://github.com/istio/istio/commit/74c6f815d449c26f2ff5584e04515670bdad442e.

Reply:
That is correct. 1.13 still uses Go 1.17.

Reply:
Good point, removed the Go CVE Statement.